salt/addons/dex/init.sls

```diff
@@ -3,8 +3,24 @@ include:
   - kubectl-config
   - kube-apiserver
 
+{% if salt.caasp_pillar.get('external_cert:dex:cert', False)
+  and salt.caasp_pillar.get('external_cert:dex:key',  False)
+%}
+
+{% from '_macros/certs.jinja' import external_pillar_certs with context %}
+
+{{ external_pillar_certs(
+      pillar['ssl']['dex_crt'],
+      'external_cert:dex:cert',
+      pillar['ssl']['dex_key'],
+      'external_cert:dex:key',
+      False
+
```

Comment:
Does this extra blank line belong?

Reply:
No, that's a missed edit after pasting stuff in from another block. Whoops.

Reply:
Removed.